WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3475,10 +3477,10 @@ e_byzantium = {
 			independent = yes
 			OR = {
 				culture = helleno_aramean
-				culture = greek
+				culture_group = byzantine
```

Comment:
While Cyprus was in its nature of a Greek connection at this time, I can see from where you are coming. Yet I do not think this change makes much sense given that the title is titular and requires the ruler to be independent, which means that it makes sense to have Greek for a Greek cultured-area.